svelte/src/components/tabs/tabs.svelte

```diff
@@ -1,49 +1,36 @@
 <script lang="ts">
   import type { Tab } from "$libs/types";
-  import { afterUpdate } from "svelte";
+  import { tabStore } from "$libs/stores";
 
-  let clazz = "";
+  const { activeTab, setActiveTab } = tabStore;
 
+  let clazz = "";
   export { clazz as class };
 
   import Button from "../button/button.svelte";
 
   export let tabs: Tab[] = [];
-  export let defaultTab: string;
-
-  let active: string;
-
-  let dirty = false;
 
-  afterUpdate(() => {
-    if (tabs.length && !active) {
-      if (!defaultTab) {
-        active = tabs[0].label;
-      } else if (!dirty) {
-        active = defaultTab;
-      }
-    }
-  });
+  $: activeTabLabel = $activeTab ?? tabs[0]?.label;
```

Comment:
if tab does not exist on package, should resort to detail tab.

or

store active tab per package?

Reply:
so this just defaults to the first tab the first time the page is loaded.  I feel like that's more robust than hardcoding the details tab, which is the first tab usually.